onnx-ecosystem/inference_demos/resnet50_modelzoo_onnxruntime_inference.ipynb

```diff
@@ -149,7 +149,12 @@
    ],
    "source": [
     "# Run the model on the backend\n",
-    "session = onnxruntime.InferenceSession('resnet50v2/resnet50v2.onnx', None)\n",
+    "# Start from ORT 1.10, ORT requires explicitly setting the providers parameter if you want to use execution providers\n",
+    "# other than the default CPU provider (as opposed to the previous behavior of providers getting set/registered by default\n",
+    "# based on the build flags) when instantiating InferenceSession.\n",
+    "# Following code assumes NVIDIA GPU is available and ORT Python package is built with CUDA. You can specify\n",
+    "# other execution providers or don't include providers parameter to use default CPU provider.\n",
+    "session = onnxruntime.InferenceSession('resnet50v2/resnet50v2.onnx', providers=['CUDAExecutionProvider'])\n",
```

Comment:
Sam comment as above... onnxruntime package is being used here

Reply:
Either 'pip install onnxruntime-gpu' or 'pip install onnxruntime', we use 'import onnxruntime' to use them, right? Correct me if I'm wrong.
This notebook uses 'import onnxruntime', so we can't really tell exact onnxruntime pakcage the system installed. But I think most of the time it's using onnxruntime package not onnxruntime-gpu package when reading the notebook tutorial.

One of the reason we want to change all the examples is that we don't want user who installs onnxruntime-gpu package and doesn't specify providers parameter: 
1.  gets warning/error message
2.  doesn't know GPU eps not actually being used since he/she doesn't specify providers.

Reply:
I will still change all the notebooks here to:
"# For example, if NVIDIA GPU is available and ORT Python package is built with CUDA, then call API as following:\n",
"# onnxruntime.InferenceSession(path/to/model, providers=['CUDAExecutionProvider']).\n",
"session = onnxruntime.InferenceSession(path/to/model)"

Since it's not confusing and users can still read the comment to know that they need to specify GPU eps if they want to use it.